### PR TITLE
Cql tests cleanup

### PR DESCRIPTION
**Pull request by mambocab**
Looking for feedback on TODOs in docstrings and (in one or 2 places) comments. I'm going to avoid expanding the scope of this PR if I can -- dealing with the TODOs will happen in a followup PR. @ptnapoleon, got time for this?

### CODE REVIEW

File: cql_tests.py [hunk 1]
```diff
@@ -113,18 +143,26 @@ def table_test(self):
...
         session.execute("CREATE INDEX testidx ON test3 (v1)")
 
-        for i in xrange(0, 10):
-            session.execute("INSERT INTO test3 (k, v1, v2) VALUES (%d, %d, %d)" % (i, i, i))
+        for i in range(0, 10):
+            session.execute("INSERT INTO test3 (k, v1, v2) VALUES ({i}, {i}, {i})".format(i=i))
 
         res = session.execute("SELECT * FROM test3 WHERE v1 = 0")
-        assert rows_to_list(res) == [[0, 0, 0]], res
+        self.assertEqual(rows_to_list(res), [[0, 0, 0]])
```
Comment (ptnapoleon) [inline cql_tests.py]: This, and others, are good candidates for being replaced with the assertions from assertions.py

File: cql_tests.py [hunk 2]
```diff
@@ -322,6 +416,12 @@ def rename_test(self):
     def invalid_string_literals_test(self):
         """
         @jira_ticket CASSANDRA-8101
+
+        - assert INSERTing into a nonexistent table fails
+        # TODO why?
```
Comment (ptnapoleon) [inline cql_tests.py]: Why would it not fail?

File: cql_tests.py [hunk 3]
```diff
@@ -374,7 +488,17 @@ def prepared_statement_invalidation_test(self):
...
+        """
+        Regression test for CASSANDRA-1337:
+
+        - CREATE a table
+        - INSERT 2 rows
+        - SELECT * from the table
+        - assert 2 rows were returned
+
+        @jira_ticket CASSANDRA-1337
+        # TODO I don't see how this is an interesting test or how it tests 1337.
```
Comment (ptnapoleon) [inline cql_tests.py]: Well it's a range scan of a small table, certainly.

**Comment by ptnapoleon** [comment]
Other than the asserts, nothing sticks out in a skim. I'll go commit by commit next.

File: cql_tests.py (hunk 2 above)
Comment (mambocab) [inline cql_tests.py]: I wasn't clear -- I understand why it should fail, but not why we have that assertion.

File: cql_tests.py [hunk 4]
```diff
@@ -322,6 +416,12 @@ def rename_test(self):
     def invalid_string_literals_test(self):
         """
         @jira_ticket CASSANDRA-8101
+
+        - assert INSERTing into a nonexistent table fails
+        # TODO why?
+        - create a table with ascii and text columns
+        - assert that trying to execute an insert statement with non-UTF8 contents raises a ProtocolException
+            - tries to insert into a nonexistent column to make sure the ProtocolException is raised over other errors
```
Comment (ptnapoleon) [inline cql_tests.py]: Looks like you tabbed too far here.

File: cql_tests.py (hunk 2 above)
Comment (ptnapoleon) [inline cql_tests.py]: Are we maybe checking that Invalid is thrown instead of Protocol?

File: cql_tests.py [hunk 5]
```diff
@@ -431,17 +573,33 @@ def local_query_test(self):
...
+
+        - populate the cluster with 2 nodes
+        - set a 1-second read timeout
+        - start one node without having it join the ring
+        - start the other node with read_iteration_delay set to 1.5 seconds
+            - (this will cause read queries to take longer than the read timeout)
+        - CREATE a table
+        - INSERT 5000 rows on a session on the node that is not a member of the ring
+        - run SELECT statements and assert they fail
+        # TODO why do queries fail? idgi
```
Comment (ptnapoleon) [inline cql_tests.py]: Not sure what you mean. We expect them to time out because the delay is longer than the timeout

File: cql_tests.py (hunk 4 above)
Comment (mambocab) [inline cql_tests.py]: No, this is intentionally a nested bullet.

File: cql_tests.py (hunk 5 above)
Comment (mambocab) [inline cql_tests.py]: Yep, that'd do it. I think this is a stale comment, I figured it out eventually.

**Comment by ptnapoleon** [comment]
I am +1

File: cql_tests.py (hunk 3 above)
Comment (mambocab) [inline cql_tests.py]: Is it worth keeping around? I don't get it.

**Comment by mambocab** [comment]
Ok, all todos are encoded as `TODO`s in the docstrings. I'm gonna merge and then work on the todos.

**Pull request merged**